casanovo/denovo/model.py

```
@@ -947,10 +957,12 @@ def predict_step(
 
         return predictions
 
-    def on_train_epoch_end(self) -> None:
+    def on_train_batch_end(self, outputs, batch, batch_idx) -> None:
```

Ruff (0.15.7)

[warning] Unused method argument: `outputs`

(ARG002)